module/tools/RoboCupConfiguration/src/RoboCupConfiguration.cpp

```diff
@@ -302,55 +320,58 @@ namespace module::tools {
     void RoboCupConfiguration::toggle_selection() {
         // Networking configuration column
         if (column_selection == 0) {
-            switch (row_selection) {
-                case 2:  // player_id
-                    player_id = player_id == 6 ? 1 : player_id + 1;
-                    break;
-                case 3:  // team_id
-                    team_id = team_id == 33 ? 1 : team_id + 1;
-                    break;
-            }
             return;
         }
         // Game configuration column
-        if (row_selection == 0) {
-            ++robot_position;
+        switch (row_selection) {
+            case 0:  // player_id
+                player_id = player_id == 6 ? 1 : player_id + 1;
+                break;
+            case 1:  // team_id
+                team_id = team_id == 33 ? 1 : team_id + 1;
+                break;
+            case 2:  // position
+                ++robot_position;
```

Comment:
I would definitely prefer for row_selection to be an instance of an enum entry or something more readable. 33's and 6's by themselves are hard to read imo